_data/blogs.yml

```diff
@@ -203,4 +203,9 @@
   img: DSC_3288.jpg
   author_name: Aman Singh
   
+- url: https://openblogsource.wordpress.com/2017/12/28/my-journey-open-source/
+  title: my-journey- open-source
+  img: opensourceblog.png
```

Comment:
Change the name of the image to the one that is of 240 x 240 pixels ie pic2.jpg

Reply:
i have added another pull request  and have deleted this image